sys-apps/tas/files/tas.tmpfiles

```diff
@@ -0,0 +1,2 @@
+z /etc/supermicro/tas.ini 0750 root root - -
```

Comment:
Why do you need to do that via tmpfiles.d rather than via ebuild?

Reply:
Whoops, you are right, this is not needed, since I am installing the *.ini anyway.